app/models/comment_like.rb

```diff
@@ -0,0 +1,6 @@
+class CommentLike < ActiveRecord::Base
+  belongs_to :comment
+  belongs_to :user
+
+  validates :user_id, :uniqueness => { :scope => :comment_id }
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)